src/txn/txn_rollback_to_stable.c

```diff
@@ -629,16 +774,44 @@ __rollback_abort_row_ondisk_kv(
  */
 static void
 __rollback_abort_newer_col_var(
```

Comment:
I don't really like `newer` in the function name. It is not clear to be it is newer to what. I think `rollback` itself is clear enough that this function is for rollback to stable.

Reply:
@kommiharibabu would it be appropriate to change all the functions with this naming style?

Reply:
Yes, You can remove all the references of `newer` function names.

Reply:
Fixed.